docs/guides/mustache.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Mustache template with tailcall
+---
+
+Whenever you have a string literal used in a directive, it should support mustache template.The mustache templates serve as placeholders, allowing you to insert dynamic content into your configurations. Simply enclose the dynamic content within double curly braces ({{ }}) wherever you need it within the directive.
+Learn more about mustache template [here](https://mustache.github.io/)
+
+### Leveraging mustache template
+
+#### Contextual Transformation
+
+To get a list of todos you will run the query `todo` defined below.
+
+> [!NOTE]
+> The base url used for this example is https://jsonplaceholder.typicode.com
+
+```graphql
+type Query {
+  todos: [Todo] @http(path: "/todos")
+}
+type Todo {
+  id: ID!
+  title: String!
+  completed: Boolean!
+}
+```
+
+Suppose you now only want the completed todos to be shown.You can accomplish this by utilizing the `completedTodos` query. This modified query includes a `completed` argument, allowing you to specify whether you want to retrieve completed or ongoing todos.
```

Comment:
There is no such thing as `completedTodos`. I think what you mean is using query params to set completed as true, say that.